managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -770,13 +770,15 @@ private synchronized void internalAsyncAddEntry(OpAddEntry addOperation) {
             currentLedgerSize += addOperation.data.readableBytes();
 
             if (log.isDebugEnabled()) {
-                log.debug("[{}] Write into current ledger lh={} entries={}", name, currentLedger.getId(),
+                log.debug("[{}] Write into current ledger lh={} entries={}", name,
+                        currentLedger == null ? null : currentLedger.getId(),
```

Comment:
For these logging cases we could add a Long  getCurrentLedgerId() method that returns null is there is no currentLedger.
We will save some code duplication